test/sim/abilities/serenegrace.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe(`Serene Grace`, function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
```

Comment:
This needs an ``after()`` hook to restore the secondary chances when the test is done. Also it would be the best practice to accordingly mod them in a ``before()`` hook.

Reply:
Very good catch! I resolved this by editing the secondary mid-event rather than modifying the actual move in Dex.

Reply:
Right, that's a better approach.